### PR TITLE
lint: Add the code inside tools/ to the checks.

### PR DESCRIPTION
**Pull request by YagoGG**
The scripts inside `tools/` weren't being checked by the linter because they don't have the `.js` extension.

They have been added to the files passed to ESLint by Gulp, and all the violations have been fixed, with the exception of two undefined elements in `tools/mockPayload` (which needs updating) and the obfuscated data in `tools/setup`.

### CODE REVIEW

File: tools/checkInactivity [hunk 1]
```diff
@@ -22,7 +22,7 @@ const inquirer = require("inquirer");
 const client = require("../src/client.js");
 const checkInactivity = require("../src/issues/checkInactivity.js");
 
-console.log("       _               _    ___                  _   _       _ _         \r\n   ___| |__   ___  ___| | _|_ _|_ __   __ _  ___| |_(___   _(_| |_ _   _ \r\n  \/ __| \'_ \\ \/ _ \\\/ __| |\/ \/| || \'_ \\ \/ _` |\/ __| __| \\ \\ \/ | | __| | | |\r\n | (__| | | |  __| (__|   < | || | | | (_| | (__| |_| |\\ V \/| | |_| |_| |\r\n  \\___|_| |_|\\___|\\___|_|\\_|___|_| |_|\\__,_|\\___|\\__|_| \\_\/ |_|\\__|\\__, |\r\n                                                                   |___\/ ");
+console.log("       _               _    ___                  _   _       _ _         \r\n   ___| |__   ___  ___| | _|_ _|_ __   __ _  ___| |_(___   _(_| |_ _   _ \r\n  / __| '_ \\ / _ \\/ __| |/ /| || '_ \\ / _` |/ __| __| \\ \\ / | | __| | | |\r\n | (__| | | |  __| (__|   < | || | | | (_| | (__| |_| |\\ V /| | |_| |_| |\r\n  \\___|_| |_|\\___|\\___|_|\\_|___|_| |_|\\__,_|\\___|\\__|_| \\_/ |_|\\__|\\__, |\r\n                                                                   |___/ ");
```
Comment (lonerz) [inline tools/checkInactivity]: Did you double check that this still looks fine?

File: tools/configLint [hunk 2]
```diff
@@ -20,10 +20,10 @@
 
 "use strict";
 
-const inquirer = require("inquirer")
+const inquirer = require("inquirer");
 const Mocha = require("mocha");
 
-console.log("                   __ _       _      _       _   \r\n                  \/ _(_)     | |    (_)     | |  \r\n   ___ ___  _ __ | |_ _  __ _| |     _ _ __ | |_ \r\n  \/ __\/ _ \\| \'_ \\|  _| |\/ _` | |    | | \'_ \\| __|\r\n | (_| (_) | | | | | | | (_| | |____| | | | | |_ \r\n  \\___\\___\/|_| |_|_| |_|\\__, |______|_|_| |_|\\__|\r\n                         __\/ |                   \r\n                        |___\/                    \n");
+console.log("                   __ _       _      _       _   \r\n                  / _(_)     | |    (_)     | |  \r\n   ___ ___  _ __ | |_ _  __ _| |     _ _ __ | |_ \r\n  / __/ _ \\| '_ \\|  _| |/ _` | |    | | '_ \\| __|\r\n | (_| (_) | | | | | | | (_| | |____| | | | | |_ \r\n  \\___\\___/|_| |_|_| |_|\\__, |______|_|_| |_|\\__|\r\n                         __/ |                   \r\n                        |___/                    \n");
```
Comment (lonerz) [inline tools/configLint]: Same here

File: tools/mockPayload [hunk 3]
```diff
@@ -29,7 +29,7 @@ const issueReferenced = require("../src/pullRequests/issueReferenced.js");
 const checkPullRequestComment = require("../src/issues/checkPullRequestComment.js");
 const travisBuildStatus = require("../src/pullRequests/travisBuildStatus.js");
 
-console.log("                       _    ____             _                 _ \r\n  _ __ ___   ___   ___| | _|  _ \\ __ _ _   _| | ___   __ _  __| |\r\n | \"_ ` _ \\ \/ _ \\ \/ __| |\/ | |_) \/ _` | | | | |\/ _ \\ \/ _` |\/ _` |\r\n | | | | | | (_) | (__|   <|  __| (_| | |_| | | (_) | (_| | (_| |\r\n |_| |_| |_|\\___\/ \\___|_|\\_|_|   \\__,_|\\__, |_|\\___\/ \\__,_|\\__,_|\r\n                                       |___\/                     ");
+console.log("                       _    ____             _                 _ \r\n  _ __ ___   ___   ___| | _|  _ \\ __ _ _   _| | ___   __ _  __| |\r\n | \"_ ` _ \\ / _ \\ / __| |/ | |_) / _` | | | | |/ _ \\ / _` |/ _` |\r\n | | | | | | (_) | (__|   <|  __| (_| | |_| | | (_) | (_| | (_| |\r\n |_| |_| |_|\\___/ \\___|_|\\_|_|   \\__,_|\\__, |_|\\___/ \\__,_|\\__,_|\r\n                                       |___/                     ");
```
Comment (lonerz) [inline tools/mockPayload]: Same here.

File: tools/mockPayload [hunk 4]
```diff
@@ -104,15 +104,15 @@ inquirer.prompt({
...
+                return answers.payloadType === "Pull request Travis build status updated (Travis webhook)";
               }
             },
             {
               type: "input",
               name: "buildURL",
               message: "Enter the URL of the Travis build logs for the pull request:",
               validate: (value) => {
-                var valid = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/.test(value);
+                var valid = /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_+.~#?&//=]*)/.test(value);
```
Comment (lonerz) [inline tools/mockPayload]: I'm not sure if I understand removing `\`?

File: tools/newRepo [hunk 5]
```diff
@@ -21,7 +21,7 @@
 const inquirer = require("inquirer");
 const client = require("../src/client.js");
 
-console.log("d8b   db d88888b db   d8b   db d8888b. d88888b d8888b.  .d88b.  \r\n888o  88 88\'     88   I8I   88 88  `8D 88\'     88  `8D .8P  Y8. \r\n88V8o 88 88ooooo 88   I8I   88 88oobY\' 88ooooo 88oodD\' 88    88 \r\n88 V8o88 88~~~~~ Y8   I8I   88 88`8b   88~~~~~ 88~~~   88    88 \r\n88  V888 88.     `8b d8\'8b d8\' 88 `88. 88.     88      `8b  d8\' \r\nVP   V8P Y88888P  `8b8\' `8d8\'  88   YD Y88888P 88       `Y88P\'                                  ");
+console.log("d8b   db d88888b db   d8b   db d8888b. d88888b d8888b.  .d88b.  \r\n888o  88 88'     88   I8I   88 88  `8D 88'     88  `8D .8P  Y8. \r\n88V8o 88 88ooooo 88   I8I   88 88oobY' 88ooooo 88oodD' 88    88 \r\n88 V8o88 88~~~~~ Y8   I8I   88 88`8b   88~~~~~ 88~~~   88    88 \r\n88  V888 88.     `8b d8'8b d8' 88 `88. 88.     88      `8b  d8' \r\nVP   V8P Y88888P  `8b8' `8d8'  88   YD Y88888P 88       `Y88P'                                  ");
```
Comment (lonerz) [inline tools/newRepo]: And here

File: tools/setup [hunk 6]
```diff
@@ -37,18 +37,20 @@ inquirer.prompt({
...
+    throw err;
+  }
   console.log("`src/config.js` already exists. Skipping.");
 }).then(() => {
   console.log("Creating `src/secrets.json` file...");
   const _0x9bc6 = ["\x32\x70\x67\x4b\x52\x66\x44\x6a\x7a\x78\x46\x34\x7a\x32\x71\x5a\x50\x6a\x69\x56\x45\x71\x35\x51\x4b\x36\x41\x53\x57\x39\x30\x73",
-  "\x68\x74\x74\x70\x73\x3a\x2f\x2f\x63\x68\x61\x74\x2e\x7a\x75\x6c\x69\x70\x2e\x6f\x72\x67",
-  "\x37\x6c\x78\x47\x79\x67\x71\x6a\x72\x59\x72\x4e\x7a\x4a\x31\x76\x6d\x78\x51\x4e\x55\x42\x4e\x46",
-  "\x7a\x75\x6c\x69\x70\x62\x6f\x74\x2d\x62\x6f\x74\x40\x63\x68\x61\x74\x2e\x7a\x75\x6c\x69\x70\x2e\x6f\x72\x67"];
+    "\x68\x74\x74\x70\x73\x3a\x2f\x2f\x63\x68\x61\x74\x2e\x7a\x75\x6c\x69\x70\x2e\x6f\x72\x67",
```
Comment (lonerz) [inline tools/setup]: Are these added spaces?

File: tools/setup [hunk 7]
```diff
@@ -69,33 +71,35 @@ inquirer.prompt({
...
   };
-  const serialized_secrets = JSON.stringify(placeholder_secrets, null, '    ');
+  const serialized_secrets = JSON.stringify(placeholder_secrets, null, "    ");
   return fs.writeFile("src/secrets.json", serialized_secrets);
 }).catch((err) => {
   throw err;
 }).then(() => {
   // Let's look for zulipbot's repo in the parent directory tree.
-  return Git.Repository.open(path.normalize(__dirname + '/..'));
+  return Git.Repository.open(path.resolve(__dirname, "../"));
```
Comment (lonerz) [inline tools/setup]: Can you explain this to me?

**Comment by YagoGG** [comment]
Closing this, as the comments have been addressed in chat.zulip.org, and the changes have been merged to master. Thanks, @lonerz!

**Pull request closed without merging**